mgtool/build.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/magefile/mage/sh"
 )
 
-func GoInstall(ctx context.Context, goPkg string, version string) (string, error) {
+func GoInstall(ctx context.Context, goPkg, version string) (string, error) {
```

Comment:
are we removing this way of writing now?

Reply:
It was auto-fixed by one of the linters